jsconf colombia (#7)
JuanCaicedo/personal-blog: content/js-conf-colombia.mdx

I had just started working as a javascript developer and wanted to learn
everything I could.

## I want to share with you why I liked it so much

That experience was so great, that I decided to make Medellín my home. Every
year I look forward JSConf Co. So much so that I turn down speaking
opportunities and fun trips to be here for it.

## A mission

I have a deep respect for the team behind JSConf Co. They have built a wonderful
conference not around personal interests, but around a mission.

It's impossible for me to say it better than Juan Pablo Buriticá and Julián
Duque did at [JSConf US
2018](https://medium.com/colombia-dev/the-butterfly-js-effect-2eeae7b32712). But
since the video for their hasn't been posted, I'll give it a shot.

Every year, JSConf Colombia provides scholarships to the conference, so that
members of the community can come learn and participate, even if they could not
otherwise afford it. They have written more about it [here](), but tldr; they
consider it a social responsibility as citizens of country with one of the
world's highest rates of inequality.

## World class content

I've learned about the [past and future of javascript]() from Brendan Eich, the

> creator of the language himself. Thorsten Lorenz, a member of the Node.js Core

Author: Thorsten isn't on the core team 😅 How about
> Thorsten Lorenz, an expert in Node.js and V8

# JS Conf Colombia is my favorite conference

The first time I came to JSConf Colombia was also my first time in Medellín. It
was October of 2015 and I had been spending some time in my hometown, Bogotá.

I was about 5 months into my first job as a javascript developer and I wanted to

> learn everything I could. When I saw a conference in Colombia, only an hour

Author: ```suggestion
learn everything I could 🤓 When I saw a conference in Colombia, only an hour
```

flight away, and that there were still tickets on sale, I decided to buy a
ticket.

It was the best conference I've ever been to! So much so that after that, I

> decided to make Medellín my home.

Author: ```suggestion
decided to make Medellín my home 😄
```

## I want to share with you why I liked it so much

Every year, [JSConf Co](https://jsconf.co/#home) is the event I most look
forward to. So much so that I turn down speaking opportunities and fun trips to
be in town for it.

I have a deep respect for the team behind JSConf Co. They have built a wonderful

> conference and community, which I think should be a model to a conferences in

Author: ```suggestion
conference and community, which I think should be a model for all conferences in
```

the technology industry.

## It has a mission

It's impossible for me to say it better than Juan Pablo Buriticá and Julián
Duque did at [JSConf US
2018](https://medium.com/colombia-dev/the-butterfly-js-effect-2eeae7b32712). But

> since the video for their hasn't been posted, I'll do what I can.

Author: ```suggestion
since the video for their hasn't been posted, I'll do what I can 😅
```

> Every year, JSConf Colombia provides many scholarships, in many forms. The idea

Author: ```suggestion
Every year, JSConf Colombia provides many scholarships, in many forms. The idea being
```

that anyone in the community should be able to participate, whether or not they
can otherwise afford it. In a country with the second highest inequality in the
world, an expensive ticket is simply out of reach for many people who could most
benefit from it.

And even without a scholarship, tickets need to be affordable, though expenses
are not. Juan Pablo wrote about this
[in
2015](https://buriti.ca/how-we-re-changing-colombia-through-open-source-communities-and-why-we-need-your-help-7825a9fd020e),
though his post is equally relevant today.

## World class content

I've learned about the [past and future of
javascript](https://www.youtube.com/watch?v=stDqtnY_iLc) from Brendan Eich himself, the
creator of the language. The following year, Thorsten Lorenz, an expert in Node.js and V8, showed us
how to [profile and solve complex memory leaks](https://youtu.be/2KnTaYk1rwg).

But It's not all serious. If you're taking a day off work, you should get to

> laugh and have fun as well.

Author: ```suggestion
laugh and have fun as well 🎉
```

I definitely have. In 2015, I watched Raquel Velez give her [first talk in
Spanish](https://youtu.be/Iqkm2j54Bks),
with an excited and supportive audience cheering her on. And in 2016, I watched Eva
Ferreira show off [how to make mind bending animations in
CSS](http://evaferreira.com.ar/jsconfco/#slide-0).

The talks at JSConf Colombia are constantly the most interesting and
thought-provoking ones I see all year, and they shape my daily work.

## Community

I have met some amazing friends because of JSConf Colombia. Friends that I see
regularly here in Medellín, and also friends that I've travelled as far as
Argentina and France to visit.

Every foreign attendee I've talked to has had the same comment about the locals
they've met at the conference. They say we're the friendliest group they've ever

> met at a conference. Most of them have fallen in love with Colombia and

Author: ```suggestion
met at a conference 😁 Most of them have fallen in love with Colombia and
```

Medellín, some of them even enough to move here.

And given that the conference has always had a focus on diversity, I've met lots
of people from all types of different backgrounds. Even from places of my own
country that I knew nothing about.

## Cultural experience

If you've never been to Colombia, this is a chance to see a unique side of our

> culture. From eating local foods like [empanadas](https://goo.gl/wxZDRT), to

Author: ```suggestion
culture. From eating local foods like [empanadas](https://goo.gl/wxZDRT) 😍 to
```
